### PR TITLE
Patch vkvg_pattern_get_color_stop_rgba and vkvg_rel_curve_to

### PR DESCRIPTION
**Pull request by Rubo3**
As per https://github.com/jpbruyere/vkvg/pull/99#issuecomment-1055811836.

### CODE REVIEW

File: src/vkvg_pattern.c [hunk 1]
```diff
@@ -172,6 +172,8 @@ vkvg_status_t vkvg_pattern_get_color_stop_rgba (VkvgPattern pat, uint32_t index,
 												float* offset, float* r, float* g, float* b, float* a) {
 	if (pat->type == VKVG_PATTERN_TYPE_SURFACE || pat->type == VKVG_PATTERN_TYPE_SOLID)
 		return VKVG_STATUS_PATTERN_TYPE_MISMATCH;
+	if (index >= 16)
+		return VKVG_STATUS_INVALID_INDEX;
 	vkvg_gradient_t* grad = (vkvg_gradient_t*)pat->data;
```
Comment (jpbruyere) [inline src/vkvg_pattern.c:177]: gradient has a count field, invalid index must be returned only if index >= grad->count

Comment (Rubo3) [reply src/vkvg_pattern.c:175]: Done

**Comment by jpbruyere** [comment]
please dont create a merge commit with master, the correct way to get master changes in your devel branch is to REBASE your branch on master, not create a merge commit.

**Review by jpbruyere — APPROVED** [review]

**Comment by jpbruyere** [comment]
your master merge commit did not affect current master history, so that's good like this, you may still do master merge on your devel branches.;-)

**Pull request merged**